openlibrary/templates/books/edit/edition.html

```diff
@@ -165,7 +165,7 @@
             </div>
```

Comment:
Leave this file for now; there's a big re-ordering that is going to be happening soon, which would result in a lot of rebase conflicts.